homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+
+const app = express();
+const port = 3000;
```

Comment:
Using all capital letters for constants, such as the port number, is a common convention in many coding styles. It helps to distinguish constants from variables. 
```suggestion
const PORT = 3000;
```